UefiPayloadPkg/Library/LbParseLib/LbParseLib.c

```diff
@@ -55,9 +55,13 @@ const UefiPayloadConfig* GetUefiPayLoadConfig() {
 
   DEBUG((DEBUG_ERROR, "Expect payload config version %016lx or %016lx, but get %016lx\n",
          UEFI_PAYLOAD_CONFIG_VERSION1, UEFI_PAYLOAD_CONFIG_VERSION2, config->Version));
+#if 0

```

Comment:
just remove if 0 and use return NULL instead?

Reply:
Sure. I wasn't certain if other platforms might have problems with that, although I guess only the linuxboot ones call these functions.